autogtp/Game.cpp

```diff
@@ -317,9 +317,17 @@ bool Game::fixSgfPlayerName(QString& weightFile) {
     QString playerName("PW[Leela Zero ");
     playerName += weightFile.left(8);
     playerName += "]";
-
     sgfData.replace(re, playerName);
 
+    if(resignation) {
+        QRegularExpression oldResult("RE\\[B\\+.*\\]");
```

Comment:
I think this should also match "W+" if white resigns when white is leading according to Tromp-Taylor, but actually has dead groups and is losing when the dead groups are removed. Maybe the current network isn't strong enough for this to happen, but I think it's possible when the network is stronger.

Reply:
I do not understand. What do you want me to do?

Reply:
Regex doesn't match if the white is leading. For example "RE[W+3.5]" doesn't match the regex and isn't replaced.